lib/workers/repository/index.js

```diff
@@ -14,6 +15,23 @@ module.exports = {
   renovateRepository,
 };
 
+async function setGitPrivateKey(config) {
+  if (config.gitPrivateKey && config.localDir) {
```

Comment:
Is there any scenario where config.localDir *wouldn't* be set?

Reply:
It's just a safety check, if by any chance it's not present the operation won't execute